docs/se/oci/03_vault.adoc

```diff
@@ -24,12 +24,24 @@
 :keywords: oci, vault
 :common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
 :feature-name: OCI Vault
+:oci-vault-javadoc: https://docs.oracle.com/en-us/iaas/tools/java/latest/com/oracle/bmc/vault/package-summary.html
+:oci-vault-url: https://docs.oracle.com/en-us/iaas/Content/KeyManagement/home.htm
+:helidon-tag: https://github.com/oracle/helidon/tree/{helidon-version}
 
 The Helidon SE OCI Vault integration provides a reactive API Oracle Cloud Vault features.
```

Comment:
I know you didn't edit this line but it doesn't parse.

Reply:
Fixed